src/Helper/Input/Checkbox.php

```diff
@@ -63,4 +68,24 @@ protected function htmlUnchecked()
 
         return $this->void('input', $attribs);
     }
+
+    protected function multiple()
+    {
+        $html = '';
+        $checkbox = clone($this);
+
+        $this->attribs['name'] .= '[]';
+
+        foreach ($this->options as $value => $label) {
+            $this->attribs['value'] = $value;
```

Comment:
that's because each checkbox has it's own value/label, Helper\Input\Radio works in the same way https://github.com/auraphp/Aura.Html/blob/2.x/src/Helper/Input/Radio.php#L52-L53